docs/specs/clients/notify/authentication.md

```diff
@@ -142,6 +142,7 @@ Paginated list of notifications with the most recently sent first. Unread notifi
   - Example: `did:web:app.example.com`
 - lmt - the max number of notifications to return. Maximum value is 50.
 - aft - the notification ID to start returning messages after. Null to start with the most recent notification
+- urf - unread first
```

Comment:
how would this work? Wouldn't this make history out of order?

Reply:
Yeah it would make it out-of-order. The idea would allow a developer to more easily show unread messages only or order unread messages first in the list if their UX desired it. E.g. a Notion notification where notifications are left unread for long periods of time.

We could probably remove it for this iteration though especially if it makes things harder.